ophasebase/tests.py

```diff
@@ -13,10 +13,10 @@ class OphaseIsActive(TestCase):
 
     def test_only_one_is_active(self):
         # create Ophase - should be active
-        o1 = Ophase.objects.create(start_date=date(2014, 4, 7), end_date=date(2014, 4, 11), is_active=True)
+        o1 = Ophase.objects.create(name="Testophase 1", is_active=True)
```

Comment:
It appears to me as if this only tests the Ophase instance itself, not any OphaseCategories. You might want to add some OphaseActiveCategories to o1.